<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Mohnish-LAPTOP" name="testRunner.testRun$TestRun" tests="7" failures="3" timestamp="2023-03-21T02:30:21 PDT" time="53.875" errors="0">
  <testcase name="runScenario" time="6.712" classname="testRunner.testRun$TestRun"/>
  <system-out/>
  <testcase name="runScenario" time="6.850" classname="testRunner.testRun$TestRun"/>
  <system-out/>
  <testcase name="runScenario" time="6.563" classname="testRunner.testRun$TestRun"/>
  <system-out/>
  <testcase name="runScenario" time="6.665" classname="testRunner.testRun$TestRun">
    <failure type="java.lang.AssertionError" message="expected [invalid] but found [password_mismatch:The two password fields didn’t match.]">
      <![CDATA[java.lang.AssertionError: expected [invalid] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at stepDefinition.RegisterExcel.user_sees_a_an_approriate_message(RegisterExcel.java:105)
at ✽.User sees a an approriate message(file:///C:/Personal/Vaish/Numpy%20Ninja/DsAlgo/Coding/DsAlgo/src/test/resources/Features/registerExcel.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="6.682" classname="testRunner.testRun$TestRun">
    <failure type="java.lang.AssertionError" message="expected [invalid] but found [password_mismatch:The two password fields didn’t match.]">
      <![CDATA[java.lang.AssertionError: expected [invalid] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at stepDefinition.RegisterExcel.user_sees_a_an_approriate_message(RegisterExcel.java:105)
at ✽.User sees a an approriate message(file:///C:/Personal/Vaish/Numpy%20Ninja/DsAlgo/Coding/DsAlgo/src/test/resources/Features/registerExcel.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="6.683" classname="testRunner.testRun$TestRun"/>
  <system-out/>
  <testcase name="runScenario" time="13.720" classname="testRunner.testRun$TestRun">
    <failure type="java.lang.AssertionError" message="expected [invalid] but found [password_mismatch:The two password fields didn’t match.]">
      <![CDATA[java.lang.AssertionError: expected [invalid] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at stepDefinition.RegisterExcel.user_sees_a_an_approriate_message(RegisterExcel.java:105)
at ✽.User sees a an approriate message(file:///C:/Personal/Vaish/Numpy%20Ninja/DsAlgo/Coding/DsAlgo/src/test/resources/Features/registerExcel.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- testRunner.testRun$TestRun -->
